client/components/cart.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { getCartFromDb, addItemToCart, removeItemFromCart } from '../store/cart'
+//I'm importing addItemToCart and adding a route for removeItemFromCart 
+//so we can increment and decrement quantity 
+
+class Cart extends React.Component {
+    constructor() {
+        super()
+        this.state = {
+            lineItems: [],
+            userId: 0
+        }
+    }
+
+    componentDidMount() {
+        const userId = this.props.match.params.id
```

Comment:
Should we consider having the url path to the component be '/cart' so an anonymous user will access it with the same path? then we can check to see if a user is logged in by looking at the store, and if so we find the user's cart, otherwise we can search for the cart by session